Formula/cryfs.rb

```diff
@@ -0,0 +1,24 @@
+class Cryfs < Formula
+  desc "Encrypts your files, so you can safely store them in Dropbox, iCloud, ..."
+  homepage "https://www.cryfs.org"
+  url "https://github.com/cryfs/cryfs.git", :tag => "0.9.7"
+  head "https://github.com/cryfs/cryfs.git", :branch => "develop"
+
+  depends_on "cmake" => :build
+  depends_on "openssl"
+  depends_on "boost"
+  depends_on "cryptopp"
+  depends_on :osxfuse
+
+  needs :cxx11
+
+  def install
+    system "cmake", ".", "-DBUILD_TESTING=off", *std_cmake_args
+    system "make", "install", "prefix=#{prefix}"
```

Comment:
Is this `prefix` really necessary? `cmake` should already know the prefix from `std_cmake_args`.